k8s/edi/Dockerfile

```diff
@@ -15,9 +15,12 @@
 #
 FROM legion/base-python-image
 
-RUN apk add --update \
-    ca-certificates curl openrc docker \
-    nginx pcre pcre-dev supervisor
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
+    python3.5 /usr/bin/add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
Python 3.5?

Reply:
apt-key doesn't work with python 3.6 so have to explicitly specify 3.5 interpreter